src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.menu {
+  margin-top: 0;
+  margin-right: 0;
+  background-color: white;
+  padding: 0;
+  border: 0;
+  display: flex;
+  justify-content: flex-end;
+  justify-content: space-between;
+}
+
+.menu:hover {
+  border: 1px solid var(--blue-color);
+}
+
+.logo {
+  margin-top: 10px;
+  margin-left: 50px;
+  float: left;
+}
```

Comment:
1. no need for those margins, you can use `align-items: center` in header to position logo
2. your `<a>` element should have width and height, so add only styles for `.header__logo-link`